segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/LogMetadata.java

```diff
@@ -15,18 +15,25 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.stream.Collectors;
+import javax.annotation.concurrent.NotThreadSafe;
 import lombok.Getter;
 import lombok.val;
 
 /**
  * Metadata for a Ledger-based log.
  */
+@NotThreadSafe
 class LogMetadata {
     //region Members
 
-    private static final byte SERIALIZATION_VERSION = 0;
+    /**
+     * Version 0: Base.
+     * Version 1: Added LedgerMetadata.Status.
+     */
+    private static final byte SERIALIZATION_VERSION = 1;
```

Comment:
Does this make the change store incompatible?

Reply:
No